ext/session/session.c

```diff
@@ -2113,7 +2113,7 @@ PHP_FUNCTION(session_set_save_handler)
 		RETURN_TRUE;
 	}
 
-	zend_error(E_DEPRECATED, "Calling session_set_save_handler() with more than 2 arguments is deprecated");
+	php_error_docref(NULL, E_DEPRECATED, "Calling session_set_save_handler() with more than 2 arguments is deprecated");
```

Comment:
Maybe
```suggestion
	php_error_docref(NULL, E_DEPRECATED, "passing individual callbacks instead of an object implementing  SessionHandlerInterface is deprecated");
```

Maybe `providing` instead of `passing` is also better.

Reply:
Sounds good. @kocsismate, what do you think?

Reply:
I went with "providing".